addons/block_code/ui/blocks/parameter_block/parameter_block.gd

```diff
@@ -18,16 +19,22 @@ var _panel_focus: StyleBox
 func _ready():
 	super()
 
-	_panel_normal = _panel.get_theme_stylebox("panel").duplicate()
-	_panel_normal.bg_color = color
-	_panel_normal.border_color = color.darkened(0.2)
+	if not definition == null and definition.variant_type == Variant.Type.TYPE_BOOL:
+		_background.visible = true
+		_background.variant = 3
```

Comment:
Please use [enumerators](https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_basics.html#enums) rather than integers. For example:

```
enum BackgroundVariant { ROUND, POINTY, ... }
```